src/components/rubric-feedback.js

```diff
@@ -0,0 +1,574 @@
+						?disabled="${this._disabled}"
+					></d2l-icon>
+					<d2l-tooltip
+						for="clear-feedback"
+						?force-show="${this._clearFeedbackInFocus}"
+						position="bottom"
+					>${this.localize('clearFeedback')}</d2l-tooltip>
+				</div>
+				${legacyEditor}
+				${invalidFeedbackTooltip}
+				<d2l-offscreen>
+					<d2l-button-subtle
+						description="${this._localizeClearFeedbackButtonDescription()}"
+						id="clear-feedback-invisible"
+						?disabled="${this._disabled}"
+						@focusin="${this._handleInvisibleFocusin}"
+						@focusout="${this._handleInvisibleFocusout}"
+						@click="${this._clearFeedbackHandler}">
+					</d2l-button-subtle>
+				</d2l-offscreen>
+			</div>
+			${editableFeedback}
+		</div>`;
+	}
+
+	shouldUpdate(changedProperties) {
+		const result = super.shouldUpdate(changedProperties);
+
+		if (changedProperties.has('enableFeedbackCopy')) {
+			const text = this.localize('criterionFeedbackWithCopy');
+			this._feedbackHeadingExtra = this.enableFeedbackCopy ? `- ${text}` : '';
+		}
+
+		if (changedProperties.has('focusFeedbackHtmlEditor')) {
+			this.dispatchEvent(new CustomEvent('focus-feedback-html-editor-changed', {
+				detail: { value: this.focusFeedbackHtmlEditor },
+				composed: true,
+				bubbles: true
+			}));
+
+			if (this.focusFeedbackHtmlEditor) {
+				// the timeout is necessary to give the htmleditor time to initialize before focusing
+				setTimeout(this._focusEditor.bind(this), 300);
+				this.focusFeedbackHtmlEditor = false;
+			}
+
+			if (changedProperties.size === 1) {
+				return false;
+			}
+		}
+
+		if (changedProperties.has('_criterionAssessment')) {
+			if (this._criterionAssessment && !this.feedbackInvalid) {
+				this.updateAssessmentFeedbackValue();
+			}
+		}
+
+		return result;
+	}
+
+	updateAssessmentFeedbackValue() {
+		if (!this._feedbackModified && !this._pendingFeedbackSaves) {
+			this.feedbackInvalid = false;
+			this._feedback = this.CriterionAssessmentHelper.getFeedbackOptimistic(
+				this._criterionAssessment,
+				this._criterionEntity,
+				this._optimisticLevelHref,
+				this.useNewBrightspaceEditor
+			);
+		}
+	}
+
+	_addFocusStylingToFeedbackWrapper() {
+		if (this.readOnly || !this.criterionAssessmentHref || !!this.compact || this.feedbackInvalid) {
+			return;
+		}
+		this.focusStyling = true;
+	}
+
+	_blurHandler(event) {
+		// prevent firing two blur events
+		event.stopPropagation();
+		this._saveFeedback(this._getEditor().html);
+	}
+
+	_canEditFeedback() {
+		if (this.readOnly || !this._criterionEntity || !this._criterionAssessment) {
+			return false;
+		}
+		return this.CriterionAssessmentHelper.canUpdateAssessment(this._criterionAssessment);
+	}
+
+	_clearFeedbackHandler() {
+		if (this._disabled) {
+			return;
+		}
+
+		this._disabled = true;
+		this._saveFeedback('').finally(() => {
+			if (!this.feedbackInvalid) {
+				this._disabled = false;
+				this.dispatchEvent(new CustomEvent('close-feedback', { bubbles: false }));
```

Comment:
All event init params default to `false`
```suggestion
				this.dispatchEvent(new CustomEvent('close-feedback'));
```